lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -111,7 +112,7 @@ def setUp(self):
         temperature_data = base_data + 273.15
         self.current_temperature_forecast_cube = set_up_variable_cube(
             temperature_data, units="Kelvin", realizations=[0, 1, 2],
-            time=time_dt, frt=frt_dt)
+            time=time_dt, frt=frt_dt, attributes=MANDATORY_ATTRIBUTE_DEFAULTS)
```

Comment:
As discussed, these attributes are added here, so that they are present when this cube is used as a template cube, for example within the `Test_process` class tests for the `ApplyCoefficientsForEnsembleCalibration` plugin. The `create_new_diagnostic_cube` function requires a `mandatory_attributes` argument, which is a dictionary containing "title", "source" and "institution" and as this function is now used as part of the `ApplyCoefficientsForEnsembleCalibration` plugin, these attributes need to be present on this cube, so that when it is used as a template cube, the attributes required by `create_new_diagnostic_cube` are present.